doc/source/operations.rst

```diff
@@ -54,8 +54,8 @@ run the following command in the root of the socok8s directory:
 
    ./run.sh remove_deployment
 
-This will delete all Helm releases, all Kubernetes resources in the ucp and
-openstack namespaces, and all persistent volumes that were provisioned for use
+This will delete all Helm releases, all Kubernetes resources in the UCP and
```

Comment:
The lack of capitalization was a deliberate choice here since the namespaces are actually 'ucp' and 'openstack'. Would it be better to leave them lower case and italicize or bold them to indicate that those terms are meant to be that way?  It's also possible that I'm being too fussy...

Reply:
fixed. lower case, literal. Not too fussy. You're correct.